tests/testthat/_snaps/to_md/table.md

```diff
@@ -6,6 +6,9 @@ title: "What have these birds been studied for? Querying science outputs with R"
 | -------- | -------------- | -------------- |  |  |
 | a        | a              | a              |  |  | 
 | a        | a              | a              |  |  | 
-|         |               |               |  |  | 
```

Comment:
One thing here is that it is now removing empty rows.  If I included a table like this

```markdown
| A column | Another column | A third column |   |   |
|----------|----------------|----------------|---|---|
| a        | a              | a              |   |   |
| a        | a              | a              |   |   |
|          |                |                |   |   |
| a        | a              | a              |   |   |
|          |                |                |   |   |
```

I would get back all but the empty rows.